Sources/Helpers/OAAppSettings.h

```diff
@@ -155,6 +155,8 @@ typedef NS_ENUM(NSInteger, EOAAutoZoomMap)
 
 @property (nonatomic, readonly) NSString *key;
 
+- (NSObject *) getProfileValue:(OAApplicationMode *)mode;
```

Comment:
Revert getProfileValue / setProfileValue and implement 
- (void) setValueFromSetting:(OAProfileSetting *)setting appMode:(OAApplicationMode *)mode;